dls-controls/malcolmjs
# Added redux timing middleware as well as adding documentation for per…

…formance

## Description

Added some middleware for capturing redux timings.
Added a page to the documentation describing how to profile the application.

## Testing instructions

Add a set up instructions describing how the reviewer should test the code
- Review code
- Check Travis build
- Review changes to test coverage
- Review the docs

## Agile board tracking

connect to #252 
closes #252
